docs/collator/guide.md

```diff
@@ -229,6 +207,49 @@ cargo build --release
 
 <!-- tabs:end -->
 
+## Registering
```

Comment:
Minor nit: I would likely only make step 1 part of the tabs and reuse the text/screenshots for Interlay as well so you don't have to enter the same text twice.

I would also add the required amount in the `collator/overview.md` file with a tab for the different networks, then you can get away without any tabs here and just reference to that overview file. If we then need to adjust the amount, we don't have to change the install instructions.